Jobs/MICRO_HERO.eai

```diff
@@ -131,7 +131,7 @@ function TeleportHome takes integer hn returns nothing
     call SetCaptainHome(BOTH_CAPTAINS, dx, dy)
     //call CaptainAttack(dx, dy)
   else
-    call CaptainGoHome()
+    call SetCaptainHome(BOTH_CAPTAINS, GetLocationX(home_location), GetLocationY(home_location))

```

Comment:
Setting captain home here will just get overridden by ARMY_TRACK so this whole section needs reworking.